### PR TITLE
split README

### PR DESCRIPTION
**Pull request by GitMensch**
information about building and debugging are now found in HACKING, along with notes on Ubuntu 20.04 and QT 5.15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,73 +128,16 @@ bug and test the latest stable version.
...
 simply delete the downloaded file. Learn more about AppImage [here](http://appimage.org/).*
 
-#### Debugging the AppImage
-
-When the AppImage crashes or is excessively slow, please provide a usable backtrace or profile run.
-To do that, you will need to download the debuginfo artifact that matches the AppImage you downloaded.
-Next, you will need to download the docker image from `ghcr.io/kdab/kdesrc-build:latest`. For that you
-need to authenticate yourself to GitHub. If you have a `ghcr.io` authentication token and use `pass`,
-you can do that with
+To find out how to debug the Appimage, see [HACKING#debugging-the-appimage].
```
Comment (milianw) [inline README.md:131]: this should be `[HACKING.md#debugging-the-appimage]`, or?

File: README.md [hunk 2]
```diff
@@ -219,73 +128,16 @@ bug and test the latest stable version.
...
-#6  0x000000000043a23e in main (argc=<optimized out>, argv=<optimized out>) at /__w/hotspot/hotspot/src/main.cpp:194
-```
+Building hotspot from source gives you the latest and greatest, but you'll have to make sure all its
+dependencies are available. Most users should probably [install hotspot](#getting-hotspot) from the distro
+package manager or as an [AppImage](#for-any-linux-distro-appimage).
 
-Note that source files are not supported in this way yet, as that would blow up the size of the docker image
-and thereby slow down our CI.
+For everyone that wants to contribute to Hotspot or use the newest version without the Appimage detailed
+notes are found at [HACKING](HACKING#table-of-contents).
```
Comment (milianw) [inline README.md:140]: dito, `(HACKING.md#table-of-contents)`

**Review by milianw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GitMensch) [reply README.md:131]: Not sure, it worked fine that way, but I'll recheck

File: README.md (hunk 2 above)
Comment (GitMensch) [reply README.md:140]: Checked, with GitHub and vscode markdown editing both work (you can "click" in the preview and on the page; language-server suggest the contained anchors if you type `[README#`.

Just specify if you want that changed and I'll do push that.

Any other changes needed / suggested?

Comment (milianw) [reply README.md:140]: if it works then I'm fine with that then

odd that it doesn't need the file extension though :shrug:

**Review by milianw — APPROVED** [review]

**Comment by milianw** [comment]
It did _not_ work but I fixed it now

**Pull request merged**